# HG changeset patch
# User Carlo de Falco <kingcrimson@tiscali.it>
# Date 1322005065 -3600
#      Wed Nov 23 00:37:45 2011 +0100
# Node ID 1398901839b62e86c074ec679c297f414ffd965a
# Parent  2af665333b865b9a976769ca2fdcbb8a750a0add
updated gnulib to get the fix for bug #34839

diff --git a/.hgsubstate b/.hgsubstate
--- a/.hgsubstate
+++ b/.hgsubstate
@@ -1,1 +1,1 @@
-8c66a845d5eaee474332d1bd8067cd355d3c67be gnulib
+ff549c078493ea86e7bd87526db276a1b94a411a gnulib
